lib/components/narrative/default/itinerary-summary.js

```diff
@@ -23,29 +36,48 @@ export default class ItinerarySummary extends Component {
       }
 
       // Add the mode icon
+      let title = leg.mode
+      if (leg.transitLeg) {
+        title = leg.routeShortName
+          ? `${leg.routeShortName}${leg.routeLongName ? ` - ${leg.routeLongName}` : ''}`
+          : leg.routeLongName
+      }
+      const style = {
+        margin: '0px',
+        padding: '3px',
+        height: '24px',
+        width: '24px',
+        backgroundColor: getModeColor(leg.mode)
+      }
+      if (i === 0) {
+        style.borderTopLeftRadius = '4px'
+        style.borderBottomLeftRadius = '4px'
+      }
+      if (i === itinerary.legs.length - 1) {
+        style.borderTopRightRadius = '4px'
+        style.borderBottomRightRadius = '4px'
+      }
       blocks.push(
-        <div key={blocks.length} className='summary-block mode-block'>
+        <div
+          style={style}
+          title={title}
+          key={blocks.length}
+          className='summary-block mode-block'
+        >
```

Comment:
Not sure where this observation should go...
The query below results in a missing bikeshare icon:

![image](https://user-images.githubusercontent.com/56846598/88728347-7ff19c80-d0ff-11ea-853a-494112277b4f.png)


http://localhost:9966/#/?ui_activeSearch=pxn1jrumg&ui_activeItinerary=3&fromPlace=600%20Columbia%20St%2C%20Orlando%2C%20FL%2C%20USA%3A%3A28.527074880317347%2C-81.38654708862306&toPlace=2521%20Dade%20Avenue%2C%20Orlando%2C%20FL%2C%20USA%3A%3A28.574572543440226%2C-81.37418746948244&date=2020-07-28&time=18%3A15&arriveBy=false&mode=WALK%2CBUS%2CRAIL&showIntermediateStops=true&maxWalkDistance=1207&optimize=QUICK&walkSpeed=1.34&ignoreRealtimeUpdates=true&companies=&bannedRoutes=&numItineraries=3&otherThanPreferredRoutesPenalty=900&preferredRoutes=

Reply:
Yea, I'm not really sure the source of this issue either. I think the bug is here: 

https://github.com/ibi-group/otp-ui/blob/c8f4f569b3c8a71bd9fd18c4f038a56ca51194b3/packages/icons/src/trimet-mode-icon.js#L28-L33

Reply:
Referenced in https://github.com/opentripplanner/otp-ui/issues/204